ui/guest/src/contentController.ts

```diff
@@ -292,9 +292,11 @@ function deleteItemFromHierarchyMap(modelId: string) {
 
 export function updateField(modelId: string, fieldId: string, index: string | number, value: unknown): void {
   const models = getCachedModels();
-  const model = { ...models[modelId] };
-  const parentModelId = getParentModelId(modelId, models, modelHierarchyMap);
-  const modelsToUpdate = collectReferrers(modelId);
+  const modelIdToEdit = isInheritedField(modelId, fieldId) ? getModelIdFromInheritedField(modelId, fieldId) : modelId;
+  const modelToEdit = { ...models[modelIdToEdit] };
+
+  const parentModelId = getParentModelId(modelIdToEdit, models, modelHierarchyMap);
+  const modelsToUpdate = collectReferrers(modelIdToEdit);
```

Comment:
Looks like `model` just got renamed to `modelToEdit` but there's not collision (i.e. unnecessary). Can we minimize the change history a bit? Basically rename back to model.
```suggestion
  const modelIdToEdit = isInheritedField(modelId, fieldId) ? getModelIdFromInheritedField(modelId, fieldId) : modelId;
  const model = { ...models[modelIdToEdit] };
  const parentModelId = getParentModelId(modelIdToEdit, models, modelHierarchyMap);
  const modelsToUpdate = collectReferrers(modelIdToEdit);
```